packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesConfiguration.java

```diff
@@ -140,6 +144,13 @@ public UpdatesConfiguration loadValuesFromMap(Map<String, Object> map) {
       mIsEnabled = isEnabledFromMap;
     }
 
+    Boolean isExpoGo = readValueCheckingType(map, "isExpoGo", Boolean.class);
```

Comment:
Let's go with `"expectsSignedManifest"` for the name of this property instead -- that more clearly communicates its effect, and will be clearer if we want to set it to true in cases other than Expo Go in the future (e.g. the dev client).